Because of the way we are building this package, the _version.py
file is never generated and hence needs to be patched in manually.

Not intended for upstream.

--- iniconfig-2.0.0/src/iniconfig/_version.py
+++ iniconfig-2.0.0/src/iniconfig/_version.py
@@ -0,0 +1,2 @@
+__version__ = version = '2.1.0'
+__version_tuple__ = version_tuple = (2, 1, 0)
